ingestion_tools/scripts/importers/visualization_precompute.py

```diff
@@ -106,6 +107,61 @@ class OrientedPointAnnotationPrecompute(PointAnnotationPrecompute):
     def neuroglancer_precompute_args(self, output_prefix: str, metadata: dict[str, Any]) -> dict[str, Any]:
         return {"is_oriented": True}
 
+    def neuroglancer_precompute(self, output_prefix: str, voxel_spacing: float) -> None:
+        # Build the oriented points
+        super().neuroglancer_precompute(output_prefix, voxel_spacing)
+
+        fs = self.config.fs
+        annotation_path = self.annotation.get_output_path()
+
+        metadata = self.annotation.metadata
+
+        # Importing this at runtime instead of compile time since zfpy (a dependency of this
+        # module) cannot be imported successfully on darwin/ARM machines.
+        import cryoet_data_portal_neuroglancer.io as io
+        from cryoet_data_portal_neuroglancer.precompute.instance_mesh import (
+            encode_oriented_mesh,
+        )
+        from cryoet_data_portal_neuroglancer.precompute.mesh import (
+            generate_mesh_from_lods,
+        )
+
+        # Convert the mesh to a precomputed format oriented mesh if a mesh file exists
+        obj_name = metadata["annotation_object"]["name"]
+
+        mesh_path = cast(OrientedPointAnnotation, self.annotation).mesh_source_path
+        if not mesh_path:
+            print(f"No mesh folder found, skipping mesh generation for {obj_name}")
+            return
+
+        mesh_path = PurePath(self.config.input_path) / mesh_path
+        local_mesh_file = fs.localreadable(f"{mesh_path}")
+
+        if fs.exists(local_mesh_file):
+            # Generates the precomputed version of the mesh in memory
+            scene = io.load_glb_file(Path(local_mesh_file))
+            oriented_mesh_at_each_lod = encode_oriented_mesh(
+                scene,
+                self.annotation.get_output_data(annotation_path),
+                max_lod=2,
+                max_faces_for_first_lod=10e6,
+                decimation_aggressiveness=5.5,
+            )
+
+            # Dump the precomputed version on the output folder
+            precompute_path = self._get_neuroglancer_precompute_path(annotation_path, output_prefix)
+            tmp_path = fs.localwritable(precompute_path)
+            oriented_mesh_path = tmp_path.replace("_orientedpoint", "_orientedmesh")
+            print(f"Generating oriented mesh for oriented point in {oriented_mesh_path}")
+            generate_mesh_from_lods(
+                oriented_mesh_at_each_lod,
+                Path(oriented_mesh_path),
+                min_mesh_chunk_dim=2,
+            )
+            fs.push(oriented_mesh_path)
+        else:
```

Comment:
I think we could remove this print, I think either this one or the one on L134 is enough